netlify.toml

```
@@ -3,7 +3,7 @@
   command = "make production-build"
 
 [build.environment]
-  HUGO_VERSION = "0.110.0"
+  HUGO_VERSION = "0.111.1"
```

```suggestion
  HUGO_VERSION = "0.111.2"
```

There's a new point release: https://github.com/gohugoio/hugo/releases/tag/v0.111.2

---

Done